yafolding.el

```diff
@@ -145,7 +145,7 @@ If given, toggle all entries that start at INDENT-LEVEL."
   "Show yafolding information of the current position."
   (interactive)
   (message "indentation: %d, indent level: %d, ingore current line: %s, element-region: %d - %d, (L%d - L%d)"
-           (yafolding--current-indentation)
+           (current-indentation)
```

Comment:
Looks like a substitution was forgotten in https://github.com/emacsorphanage/yafolding/commit/24d792053a0d4099e4ebec5a4f015b0c2a1cbf29.